fylesdk/fyle_platform_sdk.py

```diff
@@ -0,0 +1,75 @@
+import json
+
+from . import exceptions
+from .apis import v1
+from .globals.configs import sdk
+from .internals.network import Network
+
+
+class FylePlatformSDK(Network):
+    """The main class which creates a connection with Fyle APIs using OAuth2 authentication (refresh token grant type).
+
+    Parameters:
+        client_id (str): Client ID for Fyle API.
+        client_secret (str): Client secret for Fyle API.
+        refresh_token (str): Refresh token for Fyle API.
+    """
+
+    TOKEN_URL = '{}/api/oauth/token'
+
+    def __init__(self, base_url, client_id, client_secret, refresh_token):
+        # store the credentials
+        self.__base_url = base_url
+        self.__client_id = client_id
+        self.__client_secret = client_secret
+        self.__refresh_token = refresh_token
+
+        self.v1 = v1
+        
+        # get the access token
+        self.update_access_token()
+        self.set_server_url()
+
+    def update_access_token(self):
+        """Update the access token."""
+
+        self.__get_access_token()
+    
+
+    def set_server_url(self):
+        """Set the Base URL in all API objects."""
+
+        sdk.config['SERVER_URL'] = self.__base_url
```

Comment:
can we use different base url for token and getting data ? 
eg: token url: accounts.fylehq.com/router/token
server url: platform.fylehq.com/v1/admin/employees

Reply:
have TOKEN_URL and SERVER_URL now